Makefile

```diff
@@ -231,3 +231,20 @@ export KUSTOMIZE=$(GOBIN)/kustomize
 else
 export KUSTOMIZE=$(shell which kustomize)
 endif
+
+oc:
+ifeq (, $(shell which oc))
+	@{ \
+	set -e ;\
+	OC_GEN_TMP_DIR=$$(mktemp -d) ;\
+	cd $$OC_GEN_TMP_DIR ;\
+	export latest_oc_client_version=$$(curl https://mirror.openshift.com/pub/openshift-v4/clients/ocp/latest/ 2>/dev/null | grep -o \"openshift-client-linux-4.*tar.gz\" | tr -d \") ;\
+	curl -JL https://mirror.openshift.com/pub/openshift-v4/clients/ocp/latest/$${latest_oc_client_version} -o oc.tar.gz ;\
+	tar -xzvf oc.tar.gz ;\
+	mv oc ${GOBIN}/oc ;\
+	rm -rf $$OC_GEN_TMP_DIR ;\
+	}
+export OCS_OC_PATH=$(GOBIN)/oc
+else
+export OCS_OC_PATH=$(shell which oc)
```

Comment:
Setting it this way now overrides the intended behavior of allowing the user to specify their own custom `oc` path (which I use regularly :wink: ). See here: https://github.com/red-hat-storage/ocs-operator/blob/main/hack/common.sh#L39

Similarly, if you don't explicit `export` the variable then it won't be available in any of the subshells called by `make`. In general I would rather see this in the `common.sh` file.